tendrl/notifier/notification/__init__.py

```diff
@@ -114,12 +107,33 @@ def __init__(self):
             )
             raise ex
 
-    def notify_alert(self, alert):
-        # TODO: Revert to object#load method of loading objects once loading of
-        # nested dicts in objects is fixed in commons.
-        NS.notification_subscriptions = etcd_read(
-            '/notification_settings/config'
-        )
-        if alert is not None:
-            for plugin in NotificationPlugin.plugins:
-                plugin.dispatch_notification(alert)
+    def _run(self):
+        while not self.complete.is_set():
+            try:
+                alerts = get_alerts()
+                # gevent.sleep(30)
+                for alert in alerts:
+                    if not eval(alert.delivery):
+                        for plugin in NotificationPlugin.plugins:
+                            plugin.dispatch_notification(alert)
+                        update_alert_delivery(alert)
+            except(
+                ValueError,
+                KeyError,
+                etcd.EtcdException
+            )as ex:
+                Event(
```

Comment:
changed please check, log_utils only can use for Messages, not for ExceptionMessages so I have replaced all messages with this function call.